timesketch/api/v1/resources.py

```diff
@@ -339,11 +342,25 @@ def post(self, sketch_id):
             for timeline in sketch.timelines:
                 tl_colors[timeline.searchindex.index_name] = timeline.color
                 tl_names[timeline.searchindex.index_name] = timeline.name
+
+            histogram = None
+            es_total_count_unfiltered = 0
+            try:
+                buckets = result[u'aggregations'][u'data_type'][u'buckets']
```

Comment:
I'm not a huge fan of strings literals used like this due to the increased chance of typos causing hidden bugs (especially with the catching of all KeyErrors), but I"m not familiar enough with the code base to have a good feel for how much of an issue this is.
Ideas:
* define constants for these strings.
* add accessor methods/properties to the result type to allow defined programmatic access to these internal components.